src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -200,6 +221,21 @@ export default {
 				});
 			}
 
+			// track cash 794 if shown
+			if (this.showFavoriteCountryRow) {
+				const favoriteCountryLoans = _get(this.$refs, 'favoriteCountries.favoriteCountryLoans');
+				if (favoriteCountryLoans !== undefined && favoriteCountryLoans.length > 0) {
+					_each(favoriteCountryLoans, (loan, loanIndex) => {
+						loanIds.push({
+							r: -1,
+							p: loanIndex + 1,
+							c: 99,
+							l: loan.id
+						});
```

Comment:
Looking forward to talking to you about how this rpcl is working. Looks pretty interesting.